src/pykeen/training/slcwa.py

```diff
@@ -93,7 +93,7 @@ def _process_batch(
         positive_batch = batch[start:stop].to(device=self.device)
 
         # Create negative samples
-        neg_samples = self.negative_sampler.sample(positive_batch=positive_batch)
+        neg_samples, neg_samples_filter = self.negative_sampler.sample(positive_batch=positive_batch)
```

Comment:
The addition of the batch_filter to the output of sample() looks pretty clean to me. Nice.